### PR TITLE
dApp staking v3 period 2 updates

### PR DESCRIPTION
**Pull request by bobo-k2**
**Pull Request Summary**

This PR contains
- dApp staking page redesign
- new voting component
- statistics for previous period
- new banners for vote and build subperiods

image

**NOTE**
Staked amount and dApp earner data is currently available for Shibuya only. Astar and Shiden data indexing is in progress 
image

**Check list**

- [ ] contains breaking changes
- [x] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

### CODE REVIEW

**Comment by Kahonnohak** [comment]
@ayumitk 

Would be great If you can highlight the section 2 when user come to staking page from project page
Screenshot 2024-06-11 at 09 31 13

**Comment by bobo-k2** [comment]
> @ayumitk
> 
> Would be great If you can highlight the section 2 when user come to staking page from project page <img alt="Screenshot 

I fixed that in commit https://github.com/AstarNetwork/astar-apps/pull/1302/commits/af7d7c94da684661ee473bbaef8ca80e9e8321c7

**Comment by bobo-k2** [comment]
@Kahonnohak I changed layout of Tier a bit (added tier threshold value) to match Figma
image

**Comment by Kahonnohak** [comment]
@bobo-k2 not the title design, but how it scrolls (to the side) it is little funny at this moment and want to have the current production scroll back. (Each tier has the box, and next box is shown on the right)

**Comment by ayumitk** [comment]
> 1. Button doesn't have colour, is the animation not working? I do remember I had issue with the button but only had a single colour on it, did not look into it at that time.
> 2. the layout of this section is not right, have you changed anything? Please check the current layout

@Kahonnohak @bobo-k2 I've fixed them in commit [db560fd](https://github.com/AstarNetwork/astar-apps/pull/1302/commits/db560fdf5620359016843f2ede2fc8f7cc90cbbe) and [8df5a04](https://github.com/AstarNetwork/astar-apps/pull/1302/commits/8df5a04e366bac4457c91a2fcd7376fd29783249).

The button animation doesn't work on mobiles, so I added the gradient background colour.

**Review by gtg7784 — APPROVED** [review]
LGTM

**Review by impelcrypto — APPROVED** [review]
LGTM

Perhaps we might want to update the words in the toast.
image

File: src/consts.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const PERIOD1_START_BLOCKS = new Map<string, number>([
```
[the remark this answers is not in the document]
Comment (bobo-k2) [reply src/consts.ts:1]: If I am not wrong in UI is added +1 at block start. No indexer changes required

**Comment by bobo-k2** [comment]
> LGTM
> 
> Perhaps we might want to update the words in the toast. 

Thank you, fixed in https://github.com/AstarNetwork/astar-apps/pull/1302/commits/06781970f096c5afcb40f61ac53dd4eab8bd88b9

File: src/consts.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (gtg7784) [reply src/consts.ts:1]: thanks!

**Review by Kahonnohak — APPROVED** [review]
LGTM!

**Pull request merged**